autoscheduler/frontend/src/components/LandingPage/SelectTerm/NavBarSelectTerm.css

```diff
@@ -0,0 +1,15 @@
+.button-container {
+    display: flex;
+    align-items: center;
+    padding-left: 1%;

```

Comment:
We don't use percentage-based padding anywhere else on the site, so why is it being used here? Also tbh I think this looks fine with no extra padding, but using a fixed pixel value would be fine:
1% padding | no padding
--|--
![image](https://user-images.githubusercontent.com/28655462/107126997-dd1d9600-6878-11eb-888f-5df0e9f15541.png) | ![image](https://user-images.githubusercontent.com/28655462/107126994-d42cc480-6878-11eb-833c-bd7946b46de3.png)

Reply:
yeah percent padding is dumb idk why I did it, but I agree without padding is fine so I'll remove it